.storybook/manager.js

```diff
@@ -4,7 +4,7 @@ import { create } from '@storybook/theming/create'
 
 addons.setConfig({
   theme: create({
-    base: 'light',
+    base: 'dark',
```

Comment:
default dark?

Reply:
Yes. This is only the storybook frame. It has nothing to do with the components.